applications/launchpad_v2/src/components/Tabs/styles.ts

```diff
@@ -1,31 +1,61 @@
+import { animated } from 'react-spring'
 import styled from 'styled-components'
 
 export const TabsContainer = styled.div`
   display: flex;
   align-items: flex-start;
   flex-direction: column;
+  position: relative;
+  white-space: no-wrap;
 `
 
-export const Tab = styled.div<{ selected?: boolean }>`
+export const TabOptions = styled.div`
   display: flex;
-  padding: 12px;
-  border-bottom: ${({ selected }) =>
-    selected ? '4px solid #9330FF' : '4px solid #fff'};
+  align-items: center;
 `
 
-export const TabOptions = styled.div`
+export const Tab = styled.button`
   display: flex;
+  padding: 8px 12px;
+  box-shadow: none;
+  border-width: 0px;
+  border-bottom: 4px solid transparent;
+  background: transparent;
+  box-sizing: border-box;
+  margin: 0px;
+  position: relative;
+  cursor: pointer;
   align-items: center;
 `
 
-export const TabsBorder = styled.div`
+export const TabSelectedBorder = styled(animated.div)`
+  position: absolute;
   height: 4px;
-  background: red;
-  width: 100%;
+  border-radius: 2px;
+  background: ${({ theme }) => theme.accent};
+  bottom: 0;
 `
 
-export const TabBorderSelection = styled.div`
-  height: 100%;
-  background: blue;
-  width: 50px;
+export const FontWeightCompensation = styled.div`
+  visibility: hidden;
+
+  & > p {
+    margin: 0;
+  }
+`
+
+export const TabContent = styled.div<{ selected?: boolean }>`
```

Comment:
is `selected` being used anywhere?

Reply:
You are right! I'm going to merge this PR and add new one fixing this.